### DISCUSSION TITLE
Corrupted Ext4 on Orin Nano

### DISCUSSION THREAD

**Discussion by cakre** [OE4T/meta-tegra#1503] [General]
Hello,

I'm currently trying to get mender working on Orin Nano and I stumbled upon an issue with the rootfs image (or any other ext4 image e.g. the one for the data partition). I have an Orin Nano with a 32Gb sdcard and I flash it through the `doflash.sh` script. If I run `dumpe2fs` on the rootfs it throws me the error `Block bitmap checksum does not match bitmap while trying to read '/dev/mmcblk1p1' bitmaps` and recommends running `e2fsck`.
If I add the `meta-mender-core` and `meta-mender-tegra-jetpack5` layers this actually results in more errors in dmesg when writing to the aforementioned partition:

![image](https://github.com/OE4T/meta-tegra/assets/163280892/22e2c86c-f30e-4a7c-93dd-d3261de787e1)

I don't know yet If I just did something wrong setting up the build or if this warrants a bug report. I already tried another host for flashing and for building just to rule that out but to no avail.

**Comment by dwalkes** [comment]
See thread at [https://matrix.to/#/!YBfWVpJwNVtkmqVCPS:gitter.im/$2fhfF612Zl41NxOno5vXnzpG7bdhs_RNrhzUF7BCgF8?via=gitter.im&via=matrix.org&via=3dvisionlabs.com](matrix) for more detail on this one, a workaround was to write without bmaptool by manually editing the make-sdcard script I believe.  Root cause is still unknown.
